src/code/z_demo.c

```diff
@@ -558,10 +564,15 @@ void Cutscene_Command_Terminator(PlayState* play, CutsceneContext* csCtx, CsCmdB
                 play->transitionType = TRANS_TYPE_INSTANT;
                 break;
             case 8:
-                gSaveContext.fw.set = 0;
+#ifdef FW_SPLIT_AGE
+                fw_backup = gSaveContext.fw_main;
+                gSaveContext.fw_main = gSaveContext.fw_secondary;
+                gSaveContext.fw_secondary = fw_backup;
+#else
+                gSaveContext.fw_main.set = 0;
+#endif
                 gSaveContext.respawn[RESPAWN_MODE_TOP].data = 0;
                 if (!GET_EVENTCHKINF(EVENTCHKINF_45)) {
-                    SET_EVENTCHKINF(EVENTCHKINF_45);
```

Comment:
How come this gets removed, and is removed in all code paths, regardless of the ifdef?

Reply:
ah I think it's my bad on this one, during my tests I set some flags and I guess I thought this one among the flags I added, gonna revert this